src/EIPTestsFiller/StateTests/stEOF/stEIP4200/CREATE_EOF1InvalidFiller.yml

```diff
@@ -0,0 +1,206 @@
+      - ':yul object "c" { code { datacopy(0, dataoffset("r"), datasize("r")) return(0, datasize("r")) } data "r" hex"ef00010100040200010006030000000000000160015d000100" }'
+      # Legacy initcode trying to deploy invalid EOF code containing RJUMPI with target PUSH/RJUMP/RJUMPI/RJUMPV/CALLF immediate
+      # - Jump to same RJUMPI immediate
+      # Code to be deployed: PUSH1(1) RJUMPI(-1) STOP - 0xef00010100040200010006030000000000000160015dffff00
+      - ':yul object "c" { code { datacopy(0, dataoffset("r"), datasize("r")) return(0, datasize("r")) } data "r" hex"ef00010100040200010006030000000000000160015dffff00" }'
+      # - Jump to another RJUMPI immediate
+      # Code to be deployed: PUSH1(1) RJUMPI(5) STOP PUSH1(1) RJUMPI(-11) STOP - 0xef0001010004020001000c030000000000000160015d00050060015dfff500
+      - ':yul object "c" { code { datacopy(0, dataoffset("r"), datasize("r")) return(0, datasize("r")) } data "r" hex"ef0001010004020001000c030000000000000160015d00050060015dfff500" }'
+      # - Jump to RJUMP Immediate
+      # Code to be deployed: PUSH1(1) RJUMPI(3) STOP RJUMP(-9) - 0xef00010100040200010009030000000000000160015d0003005cfff7
+      - ':yul object "c" { code { datacopy(0, dataoffset("r"), datasize("r")) return(0, datasize("r")) } data "r" hex"ef00010100040200010009030000000000000160015d0003005cfff7" }'
+      # - Jump to PUSH Immediate
+      # Code to be deployed: PUSH1(1) RJUMPI(-4) STOP - 0xef00010100040200010006030000000000000160015dfffc00
+      - ':yul object "c" { code { datacopy(0, dataoffset("r"), datasize("r")) return(0, datasize("r")) } data "r" hex"ef00010100040200010006030000000000000160015dfffc00" }'
+      # - Jump to RJUMPV immediate
+      # Code to be deployed: PUSH1(1) RJUMPI(5) STOP PUSH1(1) RJUMPV(0) STOP - 0xef0001010004020001000d030000000000000160015d00050060015e01000000
+      - ':yul object "c" { code { datacopy(0, dataoffset("r"), datasize("r")) return(0, datasize("r")) } data "r" hex"ef0001010004020001000d030000000000000160015d00050060015e01000000" }'
+      # - Jump to CALLF immediate
+      # Code to be deployed: 0xef0001010008020002000d000603000000000000010000000260015d00050060015e010000006001600155b1
+      #   Code[0]: PUSH1(1) RJUMPI(2) CALLF(1) STOP
+      #   Code[1]: PUSH1(1) PUSH1(1) SSTORE RETF
+      - ':yul object "c" { code { datacopy(0, dataoffset("r"), datasize("r")) return(0, datasize("r")) } data "r" hex"ef0001010008020002000d000603000000000000010000000260015d00050060015e010000006001600155b1" }'
+      # Legacy Initcode trying to deploy invalid EOF code containing RJUMPV with count 0, and zero immediates
+      # Code to be deployed: PUSH1(1) RJUMPI( - 0xef0001010008020002000d000603000000000000010000000260015d00050060015e010000006001600155b1
+      - ':yul object "c" { code { datacopy(0, dataoffset("r"), datasize("r")) return(0, datasize("r")) } data "r" hex"ef0001010008020002000d000603000000000000010000000260015d00050060015e010000006001600155b1" }'
```

Comment:
This seems to have a second code section that is unnecessary.
(same for the next case below)

Reply:
Actually this looks to be copy-paste from the previous test "Jump to CALLF immediate", so this is wrong.

And the same for the next case below.